### ISSUE TITLE
I can't play songs [Spotify Player]...

### ISSUE THREAD

**Issue by Dv-Joan** [raycast/extensions#9860]
### Extension

https://raycast.com/mattisssa/spotify-player

#### Error:
```
Error: No active device found. Spotify is not installed.
    at Bn (/Users/joan/.config/raycast/extensions/320f40ef-a633-415a-ab0e-1e99515478f7/search.js:59:1352)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async n (/Users/joan/.config/raycast/extensions/320f40ef-a633-415a-ab0e-1e99515478f7/search.js:61:493)
```

#### Error:
```
Error: request to https://api.spotify.com/v1/me failed, reason: socket hang up
    at n0 (/Users/joan/.config/raycast/extensions/320f40ef-a633-415a-ab0e-1e99515478f7/search.js:65:4183)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

### Steps To Reproduce

1. Install the Spotify Player extension
2. Sign in on your account
3. Play the song 

*All without having the Spotify app installed*

### Current Behaviour

Is not playing te song

### Expected Behaviour

It should play the song

**Comment by jatindotdev** [comment]
duplicate of #9649, will be fixed in #9682, you can track the pr for the fix

@raycastbot close this issue

**Comment by Dv-Joan** [comment]
> duplicate of #9649, will be fixed in #9682, you can track the pr for the fix
> 
> @raycastbot close this issue

Got it , so I need to have installed Spotify app right ?

**Comment by jatindotdev** [comment]
> > duplicate of #9649, will be fixed in #9682, you can track the pr for the fix
> > 
> > @raycastbot close this issue
> 
> Got it , so I need to have installed Spotify app right ?

yes if you want to play it on your spotify mac app, but if you want to control another clients like a daemon service etc you can select that device using `Select Device` command it lists your devices you can control and play music in
